style.css

```diff
@@ -459,3 +459,177 @@ form {
   width: 134px;
   margin-top: 32px;
 }
+
+@media (min-width: 768px) {
+  .hamberger-btn {
+    display: none;
+  }
```

Comment:
- [ ] In accordance to best practises, kindly avoid code repetition. Hint: You repeated this line of code in line 577 and 632. ℹ️